### PR TITLE
Add PlayerToggleSneakingEven and PlayerToggleSprintingEvent

### PR DESCRIPTION
**Pull request by kitskub**
Signed-off-by: Jack Huey kitskub@gmail.com

### CODE REVIEW

File: src/main/java/org/spout/vanilla/event/player/PlayerToggleSprintingEvent.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	@Override
+	public void setCancelled(boolean cancelled) {
+		super.setCancelled(cancelled);
+	}
+
+	/**
+	 * Get the GameMode the player will have.
+	 * @return The GameMode the player will have
+	 */
+	public boolean isSprinting() {
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/event/player/PlayerToggleSprintingEvent.java]: You should be able to set this. (Same for sneaking)

Comment (Zidane) [inline src/main/java/org/spout/vanilla/event/player/PlayerToggleSprintingEvent.java]: ^ plus your javadocs are wrong.

Comment (kitskub) [inline src/main/java/org/spout/vanilla/event/player/PlayerToggleSprintingEvent.java]: Why should you be able to set it? Just cancel it?

**Comment by Sleaker** [comment]
Are you sure this compiles? PlayerEvent isn't cancellable and doesn't have the method setCancelled() I'm not sure how this works.  Events that implement cancellable need to actually have isCancelled defined within them.  You can't call a super method that doesn't exist and it's not explicitly defined in PlayerEvent, or in Cancellable (interface).

**Comment by kitskub** [comment]
Yes it compiles. Check EVERY other cancellable event if you don't believe me.

**Comment by Sleaker** [comment]
Ah I see, the code is boxed into Event... odd.

**Pull request merged**